assets/wizards/popups/components/popup-action-card/index.js

```diff
@@ -18,6 +18,15 @@ import { ActionCard, Button, CategoryAutocomplete } from '../../../../components
 import PopupPopover from '../popup-popover';
 import './style.scss';
 
+const placementForPopup = ( { options: { placement } } ) =>
+	( {
+		center: 'Center Overlay',
+		top: 'Top Overlay',
+		bottom: 'Bottom Overlay',
+		inline: 'Inline',
+		above_header: 'Above Header',
```

Comment:
Cann we have translatable strings here?

Reply:
Whoops. Added here: https://github.com/Automattic/newspack-plugin/pull/793/commits/a386ccd6b58080fe6bb7d25b4ef14f0eaa700ade

Reply:
Awesome, thank you